js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/Conditions.tsx

```diff
@@ -157,7 +172,10 @@ export const ConditionsNoPartitions = ({
           }
         />
       </CollapsibleSection>
-      <CollapsibleSection header="Discard conditions met">
+      <CollapsibleSection
+        header="Discard conditions met"
+        details="These conditions block materializations, and the asset won't be materialized unless future materialization conditions occur."
```

Comment:
this one's a little silly, at least with the present denominator the rate limit will never affect non partitioned assets. Should we hide it?

Reply:
I like the idea of hiding this for unpartitioned assets

Reply:
https://github.com/dagster-io/dagster/pull/15403